### PR TITLE
Fix flakiness in test_unit_studio_view

### PR DESCRIPTION
**Pull request by Muddasser**

### CODE REVIEW

File: jenkins/end_to_end_tests.sh [hunk 1]
```diff
@@ -29,4 +29,4 @@ paver install_pages > log/paver_install_pages.log
 export DISPLAY=:1
 
 # Run the tests
-paver e2e_test --with-flaky --exclude=whitelabel
+paver e2e_test common/test_html_component.py:StudioViewTest.test_unit_studio_view --with-flaky
```
Comment (jzoldak) [inline jenkins/end_to_end_tests.sh]: left over from debugging?

File: regression/tests/common/test_html_component.py [hunk 2]
```diff
@@ -172,7 +172,7 @@ def setUp(self):
         """
         super(StudioViewTest, self).setUp()
 
-    @flaky  # TODO: See https://openedx.atlassian.net/browse/LT-65
+    @flaky(max_runs=30, min_passes=30)
```
Comment (jzoldak) [inline regression/tests/common/test_html_component.py]: leftover from debugging?

**Comment by raeeschachar** [comment]
@benpatterson @jzoldak 
Please review.

**Comment by jzoldak** [comment]
@raeeschachar it looks like the merge conflict is for the flaky decorator on the other (wordcloud) test. May be unneeded, fixed in #174.

**Pull request merged**